x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,718 @@
+[[update-node-certs]]
+== Updating node security certificates
+You might need to update your TLS certificates if your current node
+certificates expire soon, you're adding new nodes to your secured cluster, or
+a security breach has broken the trust of your certificate chain.
+
+In instances where you have access to the original Certificate Authority (CA) key and certificate that you used to sign your existing node certificates (and where your CA has not been compromised), you can
+<<update-node-certs-same,use that CA to sign the new certificates>>.
+
+If you have to trust a new CA from your organization, or you need to generate a new CA yourself, you need to use this new CA to sign the new node certificates and instruct your nodes to trust the new CA. In this case, you'll
+<<update-node-certs-different,generate a new node certificate and key>> for each
+node that is signed by the new CA certificate and instruct your nodes to trust
+this certificate chain.
```

Comment:
I find this confusing, it makes it sound like you need to generate new keys and certificates for each node _only_ in this case, which is not true.  What is different between the two cases is what signs these new certificates , can we focus on that instead ?

Reply:
I'll update this sentence to:
>In this case, you'll <<update-node-certs-different,sign node certificates with your new CA>> and instruct your nodes to trust this certificate chain.